bigbone/src/main/kotlin/social/bigbone/api/method/PushNotificationMethods.kt

```diff
@@ -0,0 +1,137 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.entity.WebPushSubscription
+import social.bigbone.api.exception.BigBoneRequestException
+
+/**
+ * Allows access to API methods with endpoints having an "api/vX/push" prefix.
+ * @see <a href="https://docs.joinmastodon.org/methods/push/">Mastodon push notification API methods</a>
+ */
+class PushNotificationMethods(private val client: MastodonClient) {
+
+    private val pushEndpoint = "/api/v1/push/subscription"
+
+    /**
+     * Specify whether to receive push notifications from all, followed, follower, or none users.
+     */
+    enum class PushDataPolicy {
+        ALL,
+
+        FOLLOWED,
+
+        FOLLOWER,
+
+        NONE
```

Comment:
With the annotations gone, we could remove the blank lines or even move them into one line.

```suggestion
        ALL,
        FOLLOWED,
        FOLLOWER,
        NONE
```

```suggestion
        ALL, FOLLOWED, FOLLOWER, NONE
```